pom.xml

```diff
@@ -2568,7 +2568,7 @@
 
         <!-- Identity REST API feature -->
         <identity.api.dispatcher.version>2.0.17</identity.api.dispatcher.version>
-        <identity.server.api.version>1.3.94</identity.server.api.version>
+        <identity.server.api.version>1.3.95-SNAPSHOT</identity.server.api.version>
```

Comment:
```suggestion
        <identity.server.api.version>1.3.94</identity.server.api.version>
```

Reply:
This is added for the tracking purpose. Will update the correct api-server version once it is released.